Userland/Utilities/cksum.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <LibCrypto/Checksum/Adler32.h>
+#include <LibCrypto/Checksum/CRC32.h>
+
+int main(int argc, char** argv)
+{
+    Vector<const char*> paths;
+    const char* opt_algorithm = nullptr;
+
+    Core::ArgsParser args_parser;
+    args_parser.add_option(opt_algorithm, "Checksum algorithm (default 'crc32') ('list' to list available algorithms)", "algorithm", '\0', nullptr);
+    args_parser.add_positional_argument(paths, "File", "file", Core::ArgsParser::Required::No);
+    args_parser.parse(argc, argv);
+
+    auto algorithm = (opt_algorithm == nullptr) ? "crc32" : StringView(opt_algorithm).to_string().to_lowercase();
```

Comment:
```suggestion
    auto algorithm = (opt_algorithm == nullptr) ? "crc32" : String(opt_algorithm).to_lowercase();
```

(`StringView::to_string()` is just a wrapper around the `String` constructor anyway)